android/engine/src/main/java/org/smartregister/fhircore/engine/util/SharedPrefConstants.kt

```diff
@@ -18,4 +18,5 @@ package org.smartregister.fhircore.engine.util
 
 const val LAST_SYNC_TIMESTAMP = "last_sync_timestamp"
 const val USER_INFO_SHARED_PREFERENCE_KEY = "USER_INFO"
+const val KEY_CLOCK_INFO_SHARED_PREFERENCE_KEY = "KEY_CLOCK_INFO"
```

Comment:
we can use same USER_INFO_SHARED_PREFERENCE_KEY and hide the keycloak fact from consumer

Reply:
Using KEY_CLOCK_INFO_SHARED_PREFERENCE_KEY to store keyclock related information in secured shared preferences.

Reply:
Can you fix the type in `KEY_CLOCK_INFO_SHARED_PREFERENCE_KEY` and `KEY_CLOCK_INFO`. Should instead read as `KEYCLOAK_INFO_SHARED_PREFERENCE_KEY` and `KEYCLOAK_INFO`